examples/dns/policy.gyro

```diff
@@ -0,0 +1,7 @@
+google::dns-policy dns-policy-example
+    name: "dns-policy-example"
+    description: "DNS Policy Example"
+    network
+        network: $(google::compute-network network-example)
```

Comment:
This network subresource should be defined within this gyro config. It should also be named contextually to this example. Something like:

```
google::compute-network network-example
    name: "policy-vpc-example"
    description: "policy-vpc-example-desc"
    routing-mode: "Regional"
end
```